lib/server/controllers/logs.controller.js

```diff
@@ -223,6 +223,42 @@ const listTagsByLogId = async (request, response, next) => {
     }
 };
 
+/**
+ * Lists all environmemts associated with a log.
+ *
+ * @param {Object} request The *request* object represents the HTTP request and has properties for the request query
+ *                         string, parameters, body, HTTP headers, and so on.
+ * @param {Object} response The *response* object represents the HTTP response that an Express app sends when it gets an
+ *                          HTTP request.
+ * @param {Function} next The *next* object represents the next middleware function which is used to pass control to the
+ *                        next middleware function.
+ * @returns {undefined}
+ */
+const listEnvironmentsByLogId = async (request, response, next) => {
+    const value = await dtoValidator(GetLogDto, request, response);
+    if (!value) {
+        return;
+    }
+
+    const log = await new GetLogUseCase()
+        .execute(value);
+
+    if (log === null) {
+        response.status(404).json({
+            errors: [
+                {
+                    status: '404',
+                    title: `Log with this id (${request.params.logId}) could not be found`,
+                },
+            ],
+        });
+    } else {
+        response.status(200).json({
+            data: log.environments,
+        });
+    }
```

Comment:
Seeing the example controller above, errors should be handled using `updateExpressResponseFromNativeError`